events/story_cycles/story_cycle_almohad_events.txt

```diff
@@ -0,0 +1,401 @@
+﻿namespace = almohad
```

Comment:
Could you add a comment to the top of the file with the version of CK3 it comes from? It could make compatching easier in the future.
For example:
```
# IRToCK3: Last updated for CK3 1.X.X
```

Preferably for other files too.

Reply:
added # IRToCK3: Last updated for CK3 1.7.2 to all the files in the common and events folders

Reply:
I guess the next thing to do is update all the files for CK3 1.8?

Reply:
Yeah, updating for 1.8 will probably be needed, but the converter will only be compatched in February, so there is a lot of time.